lib/active_fedora/attributes.rb

```diff
@@ -215,31 +218,46 @@ def warn_duplicate_predicates new_name, new_properties
         end
       end
 
-      def find_or_create_defined_attribute(field, dsid, args)
-        delegated_attributes[field] ||= DelegatedAttribute.new(field, dsid, datastream_class_for_name(dsid), args)
+      # @param [Symbol] field the field to find or create
+      # @param [Class] klass the class to use to delegate the attribute (e.g. 
+      #                ActiveTripleAttribute, OmAttribute, or RdfDatastreamAttribute)
+      # @param [Hash] args 
+      # @option args [String] :delegate_target the path to the delegate
+      # @option args [Class] :klass the class to create
+      # @option args [true,false] :multiple (false) true for multi-value fields
+      # @option args [Array<Symbol>] :at path to a deep node 
+      # @return [DelegatedAttribute] the found or created attribute
+      def find_or_create_defined_attribute(field, klass, args)
```

Comment:
It appears that this method is only being used for create and not find anymore.  Should the find aspect be removed?